src/components/BulkActionsConfirmationModal/BulkActionsConfirmationModal.tsx

```diff
@@ -0,0 +1,58 @@
+import { ReactNode } from 'react'
+import { BaseConfirmationModal } from '../BaseConfirmationModal/BaseConfirmationModal'
+import { Span } from '../ui/Typography/Text'
+import { VStack } from '../ui/Stack/Stack'
+import { capitalizeFirstLetter } from './utils'
+
+interface BulkActionsConfirmationModalProps {
+  isOpen: boolean
+  onOpenChange?: (isOpen: boolean) => void
+  itemCount: number
+  actionLabel: string
+  itemLabel: string
+  descriptionLabel?: string
+  onConfirm: () => void | Promise<void>
+  confirmLabel: string
+  cancelLabel: string
+  confirmDisabled?: boolean
+  hideDescription?: boolean
+  children?: ReactNode
```

Comment:
```suggestion
  isOpen: boolean
  itemCount: number
  actionLabel: string
  itemLabel: string
  descriptionLabel?: string
  confirmLabel: string
  cancelLabel: string
  confirmDisabled?: boolean
  hideDescription?: boolean
  onConfirm: () => void | Promise<void>
  onOpenChange?: (isOpen: boolean) => void
  children?: ReactNode
```

Small nit:

- I personally prefer (not forced) to keep my properties together, then my behaviors, then maybe the `children?: ReactNode` at the very last. I feel like it improves readability by grouping similar things together.

Reply:
Gotcha, like this? 
<img width="622" height="336" alt="image" src="https://github.com/user-attachments/assets/1c1fae2e-4ca6-4287-9a2f-5706599bb4f1" />

Reply:
Yes :)